### ISSUE TITLE
auto calculated boundary in large file uploading in asp.net core and testing with bombardier

### ISSUE THREAD

**Issue by Amit-limbasiya** [codesenberg/bombardier#99]
### What version of bombardier are you using?
Hash of the commit, like 

[00d7965d6cae34c62042abb0f6c45c45b870dcf3](https://github.com/codesenberg/bombardier/commit/00d7965d6cae34c62042abb0f6c45c45b870dcf3)

### What operating system and processor architecture are you using (if relevant)?
`windows/amd64`

### What did you do?
i have written a code in asp.net core in minimal api as:

app.MapPost("/uploadFile", async (HttpContext context) => {
    string[] permittedExtensions = { ".pdf", ".bin", ".txt" };
    Console.WriteLine("Body:", context.Request.Body);
    var boundary = HeaderUtilities.RemoveQuotes(MediaTypeHeaderValue
        .Parse(context.Request.ContentType).Boundary).Value;
    foreach(var header in context.Request.Headers)
        Console.WriteLine(header);
    var reader = new MultipartReader(boundary, context.Request.Body, 524288000);
    var section = await reader.ReadNextSectionAsync();
    while (section != null)
    {
        if (ContentDispositionHeaderValue.TryParse(section.ContentDisposition, out var contentDisposition))
        {
            var fileExt = Path.GetExtension(contentDisposition.FileName.Value);
            Console.WriteLine(fileExt);
            if (string.IsNullOrEmpty(fileExt) || !permittedExtensions.Contains(fileExt) || !permittedExtensions.Contains(fileExt.ToLower()))
            {
                return Results.Problem(statusCode:415,title:"Invalid Extension");
            }
            if (contentDisposition.FileName.HasValue)
            {
                var fileName = Guid.NewGuid().ToString()+ contentDisposition.FileName.Value;
                Console.WriteLine(fileName);
                using (var stream = new FileStream(Path.Combine(Path.GetTempPath(),fileName), FileMode.Create))
                {
                    await section.Body.CopyToAsync(stream);
                    await stream.FlushAsync();
                }
            }
        }
        
        section = await reader.ReadNextSectionAsync();
    }
    return Results.Ok("File uploaded successfully."); 
});

### What actually happened?
when i tried to sent the request from postman it has header conf. like 
Content-Type: multipart/form-data; boundary=<calculated when request is sent>
Content-Length:<calculated when request is sent>
Host:<calculated when request is sent>

and it will upload the file successfully at temp folder.

but when i tried it with bombardier as:
bombardier -c 1 -n 1 -m POST --header="Content-Type: multipart/form-data; boundary= <calculated when request is sent>" --header="Content-Length:524288000"  -f "Amit Limbasiya_Form12BB_Report.pdf" "https://localhost:7246/uploadFile" --http1.

it is not working and gives error like 
The header contains invalid values at index 0: 'multipart/form-data; boundary= <calculated when request is sent>'

how to solve this problem?
basically, I am taking the pdf file as an input from user and wants to upload at my temp.folder and when i run on postman it will take random boundary automatically, how to made it work on bombardier that it will take random boundary and upload fle as postman.

**Comment by codesenberg** [comment]
Another user previously encountered similar issue in #21. In short, currently you'll have to craft the body yourself.
It should be possible to implement a feature that would provide something similar to [what cURL offers](https://everything.curl.dev/http/multipart#sending-such-a-form-with-curl).

**Comment by Amit-limbasiya** [comment]
thanks @codesenberg, just need one confirmation for what i have understood, currently i have to craft body manually to use bombardier but curl set auto-boundary and wrap the content of file with the boundary. right?

**Comment by codesenberg** [comment]
That's right.
